### PR TITLE
make sure to preserve order of account

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: extension/src/background/messageListener/popupMessageListener.ts [hunk 1]
```diff
@@ -429,30 +429,32 @@ export const popupMessageListener = (request: Request) => {
...
-            unlockedAccounts.push({
-              publicKey,
-              name: accountNameList[keyId] || `Account ${keyIdList.length}`,
-              imported,
-            });
-          }
-        }),
-      );
+
+      // eslint-disable-next-line no-plusplus
```
Comment (piyalbasu) [inline extension/src/background/messageListener/popupMessageListener.ts:433]: all that async iteration talk got me thinking maybe I should ensure order here afterall just to account for any weird edge cases :)

Comment (quietbits) [reply extension/src/background/messageListener/popupMessageListener.ts:434]: Sweet! 🙂 Simple `for` loop for the win! 👍

File: extension/src/background/messageListener/popupMessageListener.ts [hunk 2]
```diff
@@ -565,6 +568,33 @@ export const popupMessageListener = (request: Request) => {
...
+  //   var arr = [prom1, prom2];
+  //   await Promise.all(
+  //     arr.map(async (num) => {
+  //       var a = await num();
+  //       hold.push(a);
+  //     }),
+  //   );
+
+  //   console.log(hold);
+  // };
```
Comment (quietbits) [inline extension/src/background/messageListener/popupMessageListener.ts:596]: Do we need to keep this?

**Review by quietbits — APPROVED** [review]

File: extension/src/background/messageListener/popupMessageListener.ts (hunk 2 above)
Comment (piyalbasu) [reply extension/src/background/messageListener/popupMessageListener.ts:596]: lol whoops - no we do not

**Pull request merged**